src/applications/vaos/appointment-list/components/ExpressCareDetailsPage.jsx

```diff
@@ -46,9 +47,9 @@ function ExpressCareDetailsPage({
 
   return (
     <PageLayout>
-      <div className="vads-u-display--block vads-u-padding-y--2p5">
-        ‹ <Link to="/">Manage appointments</Link>
-      </div>
+      <Breadcrumbs>
+        <Link to="/">Appointment detail</Link>
```

Comment:
Might make sense to call this one "Request detail"

Reply:
Good catch. I took the file name for its face value. I should have looked more closely into this detail page to examine that its an express care request detail and not appointment detail page.